centreon/tests/e2e/features/Cloud-notifications/common.ts

```diff
@@ -180,11 +215,342 @@ const waitUntilLogFileChange = (): Cypress.Chainable => {
   );
 };
 
+let servicesFoundStepCount = 0;
+
+const stepWaitingTime = 250;
+const pollingCheckTimeout = 60000;
+const maxSteps = pollingCheckTimeout / stepWaitingTime;
+
+interface MonitoredService {
+  acknowledged?: boolean | null;
+  inDowntime?: boolean | null;
+  output?: string;
+  status?: string;
+  statusType?: string;
+}
+
+const checkServices = ({
```

Comment:
why you don't use `checkServices ` ?
you can update this function by replacing `=` by `LIKE` : https://github.com/centreon/centreon/blob/develop/centreon/tests/e2e/commons.ts#L153

Reply:
@kduret modifying `checkServicesAreMonitored` could potentially cause errors in other tests. I suspect it's being used elsewhere.

Reply:
`LIKE` is functionaly the same as `=` when used without `_` or `%`
so you should update `checkServicesAreMonitored`
if an other e2e test is impacted, you should adapt it (ex: replace `_` by `\_`)